client/lib/domains/email-forwarding/has-duplicated-email-forwards.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * @param newEmailForward a string representing a new email forward
+ * @returns { boolean } If the email forward is present in the existing email forwards collection
+ */
+export function hasDuplicatedEmailForwards( newEmailForward, existingEmailForwards ) {
+	return existingEmailForwards?.some( ( forward ) => forward.mailbox === newEmailForward );
```

Comment:
Nit: It may be worth supplying a boolean for the fall-through case where `existingEmailForwards` is not an array. (Can you tell I've been writing TypeScript recently? ;) )
```suggestion
	return existingEmailForwards?.some( ( forward ) => forward.mailbox === newEmailForward ) ?? false;
```